generators/client/templates/angular/src/main/webapp/app/admin/configuration/_configuration.component.html

```diff
@@ -3,12 +3,12 @@ <h2 jhi-translate="configuration.title">Configuration</h2>
 
     <span jhi-translate="configuration.filter">Filter (by prefix)</span> <input type="text" [(ngModel)]="filter" class="form-control">
     <label>Spring configuration</label>
-    <table class="table table-condensed table-striped table-bordered table-responsive"
+    <table class="table table-sm table-striped table-bordered"
```

Comment:
This is my bad. I removed table-responsive because it would cause some strange issues (table not taking 100% of the width or overflow in some modals) but I misread the doc and actually I just have to wrap the table inside a table-responsive !